extension/src/popup/basics/BackButton/index.tsx

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import { Icon } from "@stellar/design-system";
+import { useHistory } from "react-router-dom";
+
+import "./styles.scss";
+
+interface BackButtonProps {
+  hasBackCopy?: boolean;
+}
+
+export const BackButton = ({ hasBackCopy }: BackButtonProps) => {
```

Comment:
🙌 
❓ can we remove the old BackButton in basics/Buttons now?

Reply:
We still have a *couple* more instances of the old one being used. My plan is to systematically go through all the views converting things, and once all the instances of the old Back Button (and old components, in general) are all replaced, I'll be deleting them!